### ISSUE TITLE
ER: `getModifiedFiles()` in `create-instruction.js` Using Wrong Parameter Causing 404 Error And Workflow Termination

### ISSUE THREAD

**Issue by tony1ee** [hackforla/website#6778]
### Emergent Requirement - Problem
I've noticed Multiple ([1](https://github.com/hackforla/website/actions/runs/8872736258/job/24357754040), [2](https://github.com/hackforla/website/actions/runs/8872224489/job/24356286153)) errors in PR GHA checks.  

Specifically, the [Add Pull Request Instructions](https://github.com/hackforla/website/actions/workflows/pr-instructions.yml) workflow. The errors seemed to have started after commit [4295425](https://github.com/hackforla/website/commit/42954259e5cffa7ef40add46541fbcfbff0e58a8) was merged in PR https://github.com/hackforla/website/pull/6685.

Looking at the error messages, 
> **Unhandled error: HttpError: Not Found** for https\://api.github.com/repos/tony1ee/website/pulls/6777/files

> **Unhandled error: HttpError: Not Found** for https\://api.github.com/repos/del9ra/website/pulls/6776/files

It looks like the workflow was accessing the [List pull requests files](https://docs.github.com/en/rest/pulls#list-pull-requests-files) Github API, but provided incorrect parameter for `owner` and `repo`: `owner` should be `hackforla` instead of `<contributor-github-handle>` and `repo` should be `website`, as the pull request is for the `hackforla/website` repo

Specifically, the following API call in `github-actions/pr-instructions/create-instruction.js`
https://github.com/hackforla/website/blob/169f31679915bb460a5efc9e3946af1919d26701/github-actions/pr-instructions/create-instruction.js#L56-L71

### Issue you discovered this emergent requirement in
- N/A

### Date discovered 
04/28/2024

### Did you have to do something temporarily
- [ ] YES
- [x] NO

### Who was involved
@tony1ee

### What happens if this is not addressed
The PR workflow [Add Pull Request Instructions](https://github.com/hackforla/website/blob/gh-pages/.github/workflows/pr-instructions.yml) will continue to get 404 Not Found Error when getting modified file list of new PRs and causing confusion.

### Resources
- [Error Example 1](https://github.com/hackforla/website/actions/runs/8872736258/job/24357754040)
- [Error Example 2](https://github.com/hackforla/website/actions/runs/8872224489/job/24356286153)
- [**Add Pull Request Instructions** Workflow](https://github.com/hackforla/website/actions/workflows/pr-instructions.yml) 
- [[GitHub API] List pull requests files](https://docs.github.com/en/rest/pulls#list-pull-requests-files)
- [[GitHub Webhooks] Webhook payload object for `pull_request` ](https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=opened#pull_request)
- 4295425 Commit that first included the code segment in question
- #6685 PR for the commit 4295425
- #5182 Original Issue that [#6685](https://github.com/hackforla/website/pull/6685) was addressing

### Recommended Action Items
- [ ] Make a new issue
- [ ] Discuss with team
- [ ] Let a Team Lead know

### Potential solutions [draft]
(As a temporary solution,  reverting that merge should prevent new errors from generated and confusing contributors before a fix for this is implemented.)

Based on [[GitHub Webhooks] Webhook payload object for `pull_request` ](https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=opened#pull_request), we should supply the `owner` and `repo` parameter with names obtained from `base` instead of `head` of the `pull_request` object, see proposed replacement for line 58 and 59:
``` Javascript
const repoName = context.payload.pull_request.base.repo.name; 
const ownerName = context.payload.pull_request.base.repo.owner.login; 
```

**Comment by t-will-gillis** [comment]
Hey @tony1ee Great catch regarding the `base` versus `head` , and thanks for the proposed replacement code for lines 58 and 59.   

@anthonypz since you worked on PR #6685 fixes 5812, please let us know if you would like to continue with the fix as well.

**Comment by t-will-gillis** [comment]
Closing this ER as resolved- changes were picked up in PR #6790 and the posting of PR instructions appears to be working (will continue to monitor)
